libparsec/crates/client/tests/unit/workspace/utils.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) async fn workspace_ops_factory(
         workspace_storage_cache_size: WorkspaceStorageCacheSize::Default,
         proxy: ProxyConfig::default(),
         with_monitors: false,
-        prevent_sync_pattern: Regex::empty(),
+        prevent_sync_pattern: Regex::from_regex_str("\\.tmp").unwrap(),
```

Comment:
Doesn't this matches `foo.tmp.bar` ?

(as a regex, I would expect something like `\\.tmp$` or `^.*\\.tmp$`)

Reply:
695cbccddb9bab1fa66450f1f87c282a3d2e7098